src/components/AssetsImagePreviewFilter/index.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { CheckBox } from '@edx/paragon';
+
+import WrappedMessage from '../../utils/i18n/formattedMessageWrapper';
+import messages from './displayMessages';
+
+const AssetsImagePreviewFilter = ({ updateImagePreview, isImagePreviewEnabled }) => (
```

Comment:
Should this be "file preview"? I guess the preview is displayed for all files, making it a file preview, but it's also intended only for image files, so... Should the name match the check box label?